google/data_source_google_compute_address_test.go

```diff
@@ -0,0 +1,77 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccDataSourceGoogleAddress(t *testing.T) {
+	t.Parallel()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
```

Comment:
Can you add the CheckDestroy: 

`CheckDestroy: testAccCheckComputeAddressDestroy,`